src/main.css

```diff
@@ -0,0 +1,34 @@
+.ku-heading {
+  font-weight: bold;
+}
+
+.ku-margin-single {
+  margin: 8px;
+}
+
+.ku-grid-container {
+  display: grid;
+}
+
+.ku-grid-template-2col {
+  grid-template-columns: max-content auto;
```

Comment:
I am not sure how this works, but I guess it must be used together with `ku-grid-item-2col-end` ?

Reply:
It's basically saying that the template to use is:

- 2 columns (as there's two specifiers here;
- The first column should have a width equal to the size of the maximum content in that column; and
- The second columns size should be automatic - meaning it'll fill the remaining size of the grid container.

You can read more about templates at: https://www.w3schools.com/css/css_grid_container.asp (Although it may not mention `max-content`, as it only covers the basics (there's actually several size specifiers - event things like fractions `fr`. Actually, here you go: https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns)